### PR TITLE
debug: fix build issue in debug lastword, fix #1735

### LINKED ISSUE #1735: [Bug Report]: 维测模块没有打印多核系统上的cpu绑定信息
### 联系方式

_No response_

### 您发现了什么问题

A bug happened!
运行环境:
固件版本:

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by jinliangli**
[Detail]
fix build issue in debug lastword

[Verified Cases]
Build Pass: <py_engine_demo>
Test Pass:  <py_engine_demo>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : 2e43eeb113463468f5b54ed1ffc1df7e98291428

**Comment by skylarCai** [comment]
auto build success,number:176

**Comment by skylarCai** [comment]
autotest success,number:1399

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

File: components/debug/package.yaml [hunk 1]
```diff
@@ -77,8 +77,9 @@ def_config:
   DEBUG_LASTWORD_RAM_ADDR: 0
```
Comment (skylarCai) [inline components/debug/package.yaml:77]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

**Pull request merged**